tools/docker/README.md

```diff
@@ -1,159 +1,79 @@
 # Using docker-compose for local development
 
-The docker-compose configuration present in this directory allows for a user to quickly setup all of chainlink's services to perform actions like integration tests, acceptance tests, and development across multiple services.
+The docker-compose configuration present in this directory allows to quickly run a local development environment.
 
 # Requirements
 
 - [docker-compose](https://docs.docker.com/compose/install/)
 
 # Using the compose script
 
-Inside the `chainlink/tools/docker` directory, there is a helper script that is included which should cover all cases of integration / acceptance / development needs acroos multiple services. To see a list of available commands, perform the following:
+Inside the `chainlink/tools/docker` directory, there is a helper script that is included
```

Comment:
is this still a part of the sentence?

Reply:
I'm updating the sentence to be more clear